content/en/case-studies/cricket-analytics.md

```diff
@@ -0,0 +1,180 @@
+  to fitting noise to detect a signal. In cricket, just like any other sport,
+  there can be a large number of variables related to tracking various numbers
+  of players on the field, their attributes, the ball and several possibilities
+  of potential actions.  The complexity of data analytics and representation is
+  directly proportional to the kind of predictive questions that are put forth
+  during analysis and are highly dependent on data representation and the
+  model.  Things get even more challenging in terms of computation, data
+  comparisons when dynamic cricket play predictions are sought such as what
+  would have happened if the batsman had hit the ball at a different angle or
+  velocity?
+
+* **Predictive Analytics Complexity**
+
+  In cricket, some of the decision making is based on questions such as "how
+  often does a batsman play a certain kind of shot if the ball delivery is of a
+  particular type", or "how does a bowler change his line and length if the
+  batsman responds to his delivery in a certain way".
+  This kind of predictive analytics query requires highly granular dataset
+  availability and the capability to synthesize data and create generative
+  models that are highly accurate.
+
+## NumPy’s Role in Cricket Analytics
+
+Sports Analytics is a thriving field. Many researchers and companies
+[use NumPy](https://adtmag.com/blogs/dev-watch/2017/07/sports-analytics.aspx)
+and other PyData packages like Scikit-learn, SciPy, Matplotlib, and Jupyter.
+in addition to latest machine learning and AI techniques.  NumPy has been used
+for various kinds of cricket related sporting analytics such as:
+
+* **Data Correlation:** data graphing and [visualization](https://towardsdatascience.com/advanced-sports-visualization-with-pandas-matplotlib-and-seaborn-9c16df80a81b)
+  provides useful insights into relationship between various datasets;
+  [causal analysis](https://amplitude.com/blog/2017/01/19/causation-correlation),
+  and [big data approaches](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4996805/)
+  are used for tactical analysis. NumPy offers the core foundation for several
+  such analyses.
+
+* **Statistical Analysis:** NumPy numerical capabilities help estimate the
+  statistical significance of observational data or match events in the context
+  of various player and game tactics, estimating the game outcome by comparison
+  with a generative or static model.
+
+* **Data Visualization:** NumPy is used as the primary
+  [numerical computing](https://www.researchgate.net/publication/336886516_Data_visualization_and_toss_related_analysis_of_IPL_teams_and_batsmen_performances)
+  engine for cricket datasets. Pandas is used for statistics and I/O.
+  Matplotlib is used as the basic visualization for players. with Seaborn used
+  as the modern visualization package for Toss related analysis as well as for
+  team and player insights.
+
+{{< figure src="/images/content_images/cs/cricket-stats.png"
```

Comment:
Okay I have reproduced this figure. Unfortunately my conclusion is that I don't believe the contents. The code in the notebook is terrible, and it's statistically highly unlikely to get this spread of toss winners/losers from a simple coin toss. I suggest to just delete this figure, and the link to the paper above.

Reply:
Done.